### DISCUSSION TITLE
Theme based on color of the app

### DISCUSSION THREAD

**Discussion by pacman00700** [florisboard/florisboard#2078] [Feedback]
I recall that this keyboard had an option as a theme. Can you add this back?

**Comment by gg-gg1** [comment]
Hi. I was actually wanted to ask for this too. Is it possible to bring back this function? This is the only reason why I still am on an older version of the app.

**Reply by patrickgold** [reply]
Both of you are correct, this was already part of FlorisBoard but then removed due to the inconsistency of the dynamically generated themes. I want to have some kind of dynamic (and maybe app-dependent) themes in FlorisBoard in the future, although to what capacity I am not sure yet.

**Reply by Glitchy-Tozier** [reply]
I'm looking forward to that re-implementation! :)

**Reply by gg-gg1** [reply]
Me too. Till then I will stick with what I have now.
